### PR TITLE
repository config and env are updated

### PR DESCRIPTION
**Pull request by vadykoo**

### CODE REVIEW

File: app/Console/Commands/DocsTranslationCommand.php [hunk 1]
```diff
@@ -63,8 +63,8 @@ public function __construct(Client $client, EntityManagerInterface $em)
         parent::__construct();
 
         $this->em = $em;
-        $this->source = new Repository($client, 'laravel', 'docs');
-        $this->translation = new Repository($client, 'LaravelRUS', 'docs');
+        $this->source = new Repository($client, config('documentation.laravel-source.user'), config('documentation.laravel-source.repository'));
```
Comment (SerafimArts) [inline app/Console/Commands/DocsTranslationCommand.php:66]: The `config` function is global, the dependency is not passed explicitly, which is a violation of the PSR-1 standard (https://www.php-fig.org/psr/psr-1/#23-side-effects) and complicates code testing. Dependency Injection should be used to read configurations. Laravel has a special "contract" interface for this.

Although there are no tests at the moment, so this is not an argument. But the code itself, in my opinion, should be an example of "how to write in Laravel for future generations", so you should not use global functions and facades. They are only valid in seeders and migrations.

File: .env.example [hunk 2]
```diff
@@ -49,8 +49,8 @@ GITHUB_HOOK_SECRET=
 
 GITHUB_ORIGINAL_DOCS_USER=laravel
 GITHUB_ORIGINAL_DOCS_REPO=docs
-GITHUB_TRANSLATED_DOCS_USER=LaravelRUS
-GITHUB_TRANSLATED_DOCS_REPO=docs
+GITHUB_DOCS_TRANSLATED_USER=LaravelRUS
+GITHUB_DOCS_TRANSLATED_REPO=docs
```
Comment (SerafimArts) [inline .env.example:53]: Yes, this is a typo: https://github.com/LaravelRUS/laravel.su/blob/master/config/documentation.php

I think it is worth correcting `GITHUB_ORIGINAL_XXX` at the same time?

File: app/Console/Commands/DocsFetchCommand.php [hunk 3]
```diff
@@ -105,11 +105,12 @@ private function loadVersions(VersionsRepository $versions): \Traversable
             $progress->setMessage($branch->getName());
             $progress->advance();
 
+            $ver = intval(substr($branch->getName(), -3, 1));
```
Comment (SerafimArts) [inline app/Console/Commands/DocsFetchCommand.php:108]: The version may still be "master". What's the point?

File: app/Console/Commands/DocsFetchCommand.php [hunk 4]
```diff
@@ -105,11 +105,12 @@ private function loadVersions(VersionsRepository $versions): \Traversable
...
             //
             // В том случае, если ветка временная (или master), то игнорируем
             // её синхронизацию.
             //
-            if (! $branch->isSyncable()) {
+            if (! $branch->isSyncable() or $ver < 8) {
```
Comment (SerafimArts) [inline app/Console/Commands/DocsFetchCommand.php:113]: Why ignore version 8? Why not do this at the configs level?

File: app/Entity/Documentation.php [hunk 5]
```diff
@@ -101,14 +101,20 @@ public function getKeywordsString(): string
      */
     public function __toString(): string
     {
-        if ($this->translation->isRendered()) {
-            return (string)$this->translation;
-        }
+        // if( $this->translation->getStatus() === \App\Entity\Documentation\Translation\Status::ACTUAL ) {
```
Comment (SerafimArts) [inline app/Entity/Documentation.php:104]: Why is this "garbage" in the source?

File: app/Http/View/Composers/YandexMetrikaComposer.php [hunk 6]
```diff
@@ -44,7 +44,8 @@ public function compose(View $view): void
 
         $view->with('metrika', (object)[
             'id'      => $id,
-            'enabled' => (bool)$id,
+            // 'enabled' => (bool)$id,
+            'enabled' => false,
```
Comment (SerafimArts) [inline app/Http/View/Composers/YandexMetrikaComposer.php:48]: Now the logic is as follows: If there is no metric ID, it is disabled. Why disable it permanently?

File: config/app.php [hunk 7]
```diff
@@ -80,7 +80,7 @@
     |
     */
 
-    'locale' => 'ru',
+    'locale' => 'uk',
```
Comment (SerafimArts) [inline config/app.php:83]: If you need an alternative locale, then it makes sense to move this to the configuration level. The current production site locale requires the russian.

File: config/menu.php [hunk 8]
```diff
@@ -12,30 +12,30 @@
 return [
     'footer' => [
         'nav'  => [
```
Comment (SerafimArts) [inline config/menu.php:14]: If a different language requires different links, then you need to redesign the settings structure, and not cut out the old ones.

File: resources/views/page/docs/partials/translation-status.blade.php [hunk 9]
```diff
@@ -2,19 +2,19 @@
 
 @if ($page->translation->getStatus() !== \App\Entity\Documentation\Translation\Status::ACTUAL)
     @if ($page->translation->getStatus() === \App\Entity\Documentation\Translation\Status::MISSING)
-        <aside class="translation-status label warning">
+        {{-- <aside class="translation-status label warning">
             <span class="fa fa-times-circle-o"></span>
-            Перевод отсутсвует
-        </aside>
+            Переклад відсутній
```
Comment (SerafimArts) [inline resources/views/page/docs/partials/translation-status.blade.php:7]: This should be taken from the i18n settings

File: resources/views/page/docs/partials/translation-status.blade.php [hunk 10]
```diff
@@ -2,19 +2,19 @@
...
+        </aside> --}}
     @elseif($page->translation->getStatus() === \App\Entity\Documentation\Translation\Status::BEHIND)
         <aside class="translation-status label notice">
             <span class="fa fa-exclamation-triangle"></span>
-            Перевод немного отстаёт от оригинала
+            Переклад трохи застарів від оригіналу
```
Comment (SerafimArts) [inline resources/views/page/docs/partials/translation-status.blade.php:12]: Same: i18n

File: resources/views/page/docs/partials/translation-status.blade.php [hunk 11]
```diff
@@ -2,19 +2,19 @@
...
         </aside>
     @else
         <aside class="translation-status label warning">
             <span class="fa fa-times-circle-o"></span>
-            Перевод сильно отстаёт от оригинала, может быть неактуален
+            Переклад сильно застарів від оригинала, може бути неактуальним для нових версій
```
Comment (SerafimArts) [inline resources/views/page/docs/partials/translation-status.blade.php:17]: Same: i18n

File: resources/views/page/docs/partials/versions.blade.php [hunk 12]
```diff
@@ -7,7 +7,7 @@
 
 <section class="container versions">
     @if ($versions->count())
-        Версия фреймворка:
+        Версія фреймворка:
```
Comment (SerafimArts) [inline resources/views/page/docs/partials/versions.blade.php:10]: Same: i18n

File: resources/views/page/docs/partials/versions.blade.php [hunk 13]
```diff
@@ -21,9 +21,9 @@
 
         <aside>
             @if(isset($version))
-                <a href="{!! route('status.show', ['version' => $version->name]) !!}">Прогресс перевода</a>
+                <a href="{!! route('status.show', ['version' => $version->name]) !!}">Прогрес перекладу</a>
             @else
-                <a href="{!! route('status') !!}">Прогресс перевода</a>
+                <a href="{!! route('status') !!}">Прогрес перекладу</a>
             @endif
         </aside>
```
Comment (SerafimArts) [inline resources/views/page/docs/partials/versions.blade.php:28]: Same: i18n

File: resources/views/page/docs/show.blade.php [hunk 14]
```diff
@@ -9,7 +9,7 @@
 @endphp
 
 @section('title'){{ $page->title }} (Laravel {{ $version->name }})@stop
-@section('description')Русская документация Laravel {{ $version->name }} - {{ $page->title }}@stop
+@section('description')Українська документація Laravel {{ $version->name }} - {{ $page->title }}@stop
```
Comment (SerafimArts) [inline resources/views/page/docs/show.blade.php:12]: Same: i18n

File: resources/views/page/home.blade.php [hunk 15]
```diff
@@ -6,19 +6,17 @@
     <section class="splash">
         <div class="container">
             <section class="splash-description">
-                <h1>Laravel — php-фреймворк нового поколения</h1>
+                <h1>Laravel — популярний php-фреймворк, що постійно оновлюється</h1>
                 <span>
-                    Мы верим, что процесс разработки только тогда наиболее
-                    продуктивен, когда работа с фреймворком приносит радость
-                    и удовольствие. Счастливые разработчики пишут лучший код.
+                    Має широку спільноту по світу. Дозволяє будувати швидкі та безпечні сайти та має чудову документацію.
```
Comment (SerafimArts) [inline resources/views/page/home.blade.php:11]: Same: i18n

File: resources/views/page/home.blade.php [hunk 16]
```diff
@@ -6,19 +6,17 @@
...
                 </span>
             </section>
 
             <section class="splash-image">&nbsp;</section>
         </div>
     </section>
 
-    <section class="section orchid">
```
Comment (SerafimArts) [inline resources/views/page/home.blade.php:21]: Requires adding site settings, depending on locale settings. You can't just cut the code. This does not satisfy the original objectives of the website.

**Review by SerafimArts — COMMENTED** [review]
@vadykoo I cannot accept the changes as they are. Too many changes that break the site.

In addition, please describe the tasks that you want to solve with this PR, since the first and second commits are not related in meaning to each other.

File: resources/views/page/status/show.blade.php [hunk 17]
```diff
@@ -12,7 +12,7 @@
     @endphp
 
     <section class="container translation-progress">
-        <h1>Прогресс перевода Laravel {{ $current->name }}</h1>
+        <h1>Прогрес перекладу Laravel {{ $current->name }}</h1>
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:15]: Same: i18n

File: resources/views/page/status/show.blade.php [hunk 18]
```diff
@@ -22,16 +22,16 @@
         @include('page.status.partials.menu')
 
         @if($current->isDocumented)
-            <span class="translation-progress-description">Перевод показывается в меню</span>
+            <span class="translation-progress-description">Переклад відображається в меню</span>
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:25]: Same: i18n

File: resources/views/page/status/show.blade.php [hunk 19]
```diff
@@ -22,16 +22,16 @@
...
         @else
-            <span class="translation-progress-description">Перевод не показывается в меню, но доступен по прямой ссылке</span>
+            <span class="translation-progress-description">Переклад не відображається в меню, але доступний по прямому посиланню</span>
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:27]: Same: i18n

File: resources/views/page/status/show.blade.php [hunk 20]
```diff
@@ -22,16 +22,16 @@
...
         @endif
 
         <table class="table-auto">
             <thead>
                 <tr>
                     <th>Страница</th>
-                    <th>Прогресс перевода</th>
+                    <th>Прогрес перекладу</th>
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:34]: Same: i18n

**Comment by vadykoo** [comment]
sorry I sent that pull by mistake

File: resources/views/page/status/show.blade.php [hunk 21]
```diff
@@ -50,12 +50,12 @@ class="translation-page-description">/{{$current->name}}/{{ $page->urn }}</a>
                     </td>
                     <td class="translation-page-status">
                         @if ($page->translation->getStatus() === \App\Entity\Documentation\Translation\Status::MISSING)
-                            <span class="label warning">Перевод отсутсвует</span>
+                            <span class="label warning">Переклад відсутній</span>
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:53]: Same: i18n

File: resources/views/page/status/show.blade.php [hunk 22]
```diff
@@ -50,12 +50,12 @@ class="translation-page-description">/{{$current->name}}/{{ $page->urn }}</a>
...
                         @elseif($page->translation->getStatus() === \App\Entity\Documentation\Translation\Status::ACTUAL)
-                            <span class="label success">Перевод актуален</span>
+                            <span class="label success">Переклад актуальний</span>
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:55]: Same: i18n

File: resources/views/page/status/show.blade.php [hunk 23]
```diff
@@ -50,12 +50,12 @@ class="translation-page-description">/{{$current->name}}/{{ $page->urn }}</a>
...
                         @else
                             <span class="label notice">
-                                Перевод отстаёт на {{$page->translation->diff}}
+                                Переклад відстає на {{$page->translation->diff}}
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:58]: Same: i18n

File: resources/views/page/status/show.blade.php [hunk 24]
```diff
@@ -69,11 +69,11 @@ class="translation-page-description">/{{$current->name}}/{{ $page->urn }}</a>
                                 {{ substr($page->translation->targetCommit, 0, 7) }}
                             </a>
                         @else
-                            Файл перевода отсутсвует
+                            Файл Перекладу відсутній
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:72]: Same: i18n

File: resources/views/page/status/show.blade.php [hunk 25]
```diff
@@ -69,11 +69,11 @@ class="translation-page-description">/{{$current->name}}/{{ $page->urn }}</a>
...
                         @endif
                         <hr />
                         @if($page->translation->commit)
-                            Последний коммит
+                            Останній комміт
```
Comment (SerafimArts) [inline resources/views/page/status/show.blade.php:76]: Same: i18n

File: resources/views/partials/footer.blade.php [hunk 26]
```diff
@@ -14,7 +14,7 @@
         @endforeach
     </section>
     <aside class="container footer-bottom">
-        <address>Русскоязычное сообщество Laravel Framework &copy; {{ date('Y') }}</address>
+        <address>Українська документація Laravel Framework &copy; {{ date('Y') }}</address>
```
Comment (SerafimArts) [inline resources/views/partials/footer.blade.php:17]: Same: i18n

In addition, please note:
1) The source code is distributed under the MIT license. You can't just go and change the copyright.
2) "Russian-speaking" ("Русскоязычное") means Ukrainian as well. We can, if you wish, add Ukrainian documentation. You will not need to change copyrights.

File: resources/views/partials/header.blade.php [hunk 27]
```diff
@@ -14,12 +14,6 @@
         <nav class="container extras-items">
             <a href="https://laravel.com/" target="_blank" rel="nofollow">Laravel</a>
             <a href="https://laravel-idea.com" target="_blank" rel="nofollow">Laravel IDEA</a>
-            <a href="https://sleepingowladmin.ru/" target="_blank" rel="nofollow">SleepingOwl</a>
```
Comment (SerafimArts) [inline resources/views/partials/header.blade.php:17]: You can't just cut links without providing the ability to include them on the main site.

File: resources/views/partials/header.blade.php [hunk 28]
```diff
@@ -36,11 +30,11 @@
                 </section>--}}
 
                 <nav class="menu-items">
-                    <a href="{{ route('home') }}" class="@active('home')">Главная</a>
```
Comment (SerafimArts) [inline resources/views/partials/header.blade.php:39]: Please use i18n

**Comment by SerafimArts** [comment]
> sorry I sent that pull by mistake

@vadykoo no problem =)

However, consider the licensing agreement (MIT) and respect the work of others. Copyright owned by the community there for a reason.

Anyway, I have no objection if you want to add an alternative Ukrainian translation and localization. On the contrary, it will only be a plus ;)

**Comment by vadykoo** [comment]
> > sorry I sent that pull by mistake
> 
> @vadykoo no problem =)
> 
> However, consider the licensing agreement (MIT) and respect the work of others. Copyright (in the footer) owned by the community there for a reason.
> 
> Anyway, I have no objection if you want to add an alternative Ukrainian translation and localization. On the contrary, it will only be a plus ;)

Thanks for the advice, so I need to so I should leave the phrase in the footer?

**Comment by SerafimArts** [comment]
> Thanks for the advice, so I should leave the phrase in the footer if I decide to create a new site based on this code?

Frankly, the MIT license simply means copying the original license file (like `LICENSE.md`). However, this file is not in this repository, so the footer is used as copyright. This happened because no one had thought about it before.

If you prefer, I can simply add this license file. Then the footer can be changed as you like. Will it be convenient for you?

**Comment by vadykoo** [comment]
> > Thanks for the advice, so I should leave the phrase in the footer if I decide to create a new site based on this code?
> 
> Frankly, the MIT license simply means copying the original license file (like `LICENSE.md`). However, this file is not in this repository, so the footer is used as copyright. This happened because no one had thought about it before.
> 
> If you prefer, I can simply add this license file. Then the footer can be changed as you like. Will it be convenient for you?

It will be great! Thanks, this site helps me a lot in understanding the framework at the beginning

**Comment by SerafimArts** [comment]
Done: https://github.com/LaravelRUS/laravel.su/commit/b13de8481b64c723b5f5e0035aa1b9b62ac7ff53

**Pull request closed without merging**